lang/en.json

```diff
@@ -24,7 +24,7 @@
         "ScheduleAnImportGA4": "Schedule An Import for Google Analytics 4 (GA4)",
         "StartDate": "Start Date",
         "CreationDate": "Web Property Creation Date",
-        "StartDateHelp": "The date to start importing from. By default, this is the day you created your GA web property.",
+        "StartDateHelp": "The date to start importing from. By default, this is the day you created your GA web property. (YYYY-MM-DD) ",
```

Comment:
This screenshot should have been updated for start_date too right with new change ?
`https://raw.githubusercontent.com/matomo-org/plugin-GoogleAnalyticsImporter/e02cce4a6f021b859b91d35981c1518c5351362a/tests/UI/expected-ui-screenshots/GoogleAnalyticsImporter_load_ua.png`

Reply:
@snake14 apart from this all okay and good to merge

Reply:
Yeah. It hasn't run the test and generated the new screenshot yet. As soon as it does, I'll update it.